@ABLITZER/plotPIs.m

```diff
@@ -0,0 +1,68 @@
+%  Filename: plotPIs.m (methods of ABLITZER class��
+%  Abstract:
+%        Plot performance index (positional/turning) of different groups (1. exp only; 2. with self-control; 3. with unpaired control)
+%
+%   SYNTAX:
+%        1. obj.plotPIs(flag1,flag2)
+%        2. plotPIs(obj,flag1,flag2)
+%
+% - plotPIs
+%      - ABLITZER method
+%      -arg1: metric type (positional/turning/shock)
+%      -arg2: (number of arguments) which groups to plot (1. exp only; 2. with self-control; 3. with unpaired control)
+%  TODO:
+
+
+function [OLcontrol,OLexp,p] = plotPIs(obj,...ABLITZAER object
+flag1,... % 1: exp only; 2: with self-control; 3: with unpaired control
```

Comment:
```suggestion
numGroups,... % 1: exp only; 2: with self-control; 3: with unpaired control
```
If possible, we should use a specific word to represent the meaning of the variable.